components/gcp-click-to-deploy/app-config.yaml

```diff
@@ -0,0 +1,57 @@
+# Just for local test
```

Comment:
Why not put this file directly under src/user_config as required? Why does the dev have to copy it before starting the app?

Reply:
When run in k8s cluster, user need to provide ```app-config.yaml``` through config map.
By not creating user_config folder, if there's a bug or something caused config map failed mounting to container, user will notice their app doesn't work, instead of the app still running, with test/dev config.